src/view/droppable-dimension-publisher/droppable-dimension-publisher.jsx

```diff
@@ -47,10 +45,12 @@ export default class DroppableDimensionPublisher extends Component {
       isDropDisabled,
       targetRef,
     } = this.props;
-    invariant(targetRef, 'DimensionPublisher cannot calculate a dimension when not attached to the DOM');
```

Comment:
why get rid of invariant here but not elsewhere in this file where it's used?

Reply:
Good call - i will look at why this was done. I think i was battling a flow issue.

Reply:
I think it has to do with flow not understanding invariant